File: 3370.py
# 3370. Smallest Number With All Set Bits

class Solution:
    def smallestNumber(self, n: int) -> int:
        result = 0
        for i in range(n + 1):
            if pow(2, i) > n:
                result = pow(2, i) - 1
                break
        return result

File: 3370.cpp
// 3370. Smallest Number With All Set Bits

#include <iostream>
#include <math.h>
using namespace std;

class Solution {
public:
    int smallestNumber(int n) {
        int result = 0;
        for(int i = 0; i <= n; i++){
            if(pow(2, i) > n){
                result = pow(2, i) - 1;
                break;
            }
        }
        return result;
    }
};